homeassistant/components/light/__init__.py

```diff
@@ -295,9 +295,7 @@ def filter_turn_on_params(light, params):
     if not supported_features & SUPPORT_WHITE_VALUE:
         params.pop(ATTR_WHITE_VALUE, None)
 
-    supported_color_modes = (
-        light._light_internal_supported_color_modes  # pylint:disable=protected-access
-    )
+    supported_color_modes = light._light_internal_supported_color_modes
```

Comment:
This looks like a false negative bug for protected-access. This function is not a method of the light or a decorator used in light instances. It just happens to have the first parameter as the light instance.

Reply:
Indeed, it looks like a regression. Will be resolved in a future update. https://github.com/PyCQA/pylint/issues/5989